### PR TITLE
[SIG-2447] Refactor AttachmentViewer component

### PR DESCRIPTION
**Pull request by janjaap**
This PR contains changes to the `IncidentDetail` internal `AttachmentViewer` component. This PR is part of a bigger exercise to refactor the `IncidentDetail` component so that it doesn't use the application's state anymore.

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js [hunk 1]
```diff
@@ -1,39 +1,69 @@
...
+  position: absolute;
+  right: 10px;
+  top: calc(50% - ${themeSpacing(4)});
+`;
+
+const Wrapper = styled.div`
+  position: relative;
+  text-align: center;
+  background-color: ${themeColor('tint', 'level2')};
+  height: 722px;
```
Comment (aditudorache) [inline src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js:31]: has this number a special meaning? If is a const could we use then a const value that has a meaningful name?

File: src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js [hunk 2]
```diff
@@ -1,39 +1,69 @@
...
+  display: flex;
+`;
+
+const Img = styled.img`
+  max-height: 722px;
```
Comment (aditudorache) [inline src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js:36]: has this number a special meaning?

File: src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js [hunk 3]
```diff
@@ -1,39 +1,69 @@
...
+
+const PreviousButton = styled(IconButton)`
+  position: absolute;
+  left: 10px;
+  top: calc(50% - ${themeSpacing(4)});
+`;
+
+const NextButton = styled(IconButton)`
+  position: absolute;
+  right: 10px;
```
Comment (aditudorache) [inline src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js:23]: themeSpacing?

File: src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js [hunk 4]
```diff
@@ -1,39 +1,69 @@
...
-import './style.scss';
+const IconButton = styled(Button).attrs({
+  size: 32,
+  iconSize: 16,
+  variant: 'application',
+})``;
+
+const PreviousButton = styled(IconButton)`
+  position: absolute;
+  left: 10px;
```
Comment (aditudorache) [inline src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js:17]: themeSpacing?

File: src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js (hunk 1 above)
Comment (janjaap) [reply src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js:31]: I've used the value that was already in that component's style. I assume that that particular value has been chosen deliberately so I left it as it was.

File: src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js (hunk 2 above)
Comment (janjaap) [reply src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js:36]: See previous comment; value was already present.

File: src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js (hunk 3 above)
Comment (janjaap) [reply src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js:23]: The button doesn't have to align to other elements, so theme spacing values here aren't necessary.

File: src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js (hunk 4 above)
Comment (janjaap) [reply src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js:17]: See previous comment.

File: src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js (hunk 1 above)
Comment (aditudorache) [reply src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js:31]: All right, I was thinking about something like:
```code 
const ATTACHEMENT_VIEWER_HEIGHT = 722;

...
   height: ${ATTACHEMENT_VIEWER_HEIGHT}px;
...
   max-height: ${ATTACHEMENT_VIEWER_HEIGHT}px;

```

**Pull request merged**